core/templates/dev/head/css/oppia.css

```diff
@@ -961,20 +961,20 @@ textarea {
     margin-left: 5vw;
     margin-right: 5vw;
     max-width: 89vw;
-    width: 632px;
+    width: initial;
   }
 }
 @media (max-width: 632px) {
   .oppia-exp-summary-tiles-container {
     margin-left: 2vw;
     margin-right: 2vw;
     max-width: 96vw;
-    width: 420px;
+    width :initial;
```

Comment:
Note that this and the next one are not necessary. They are subsumed by the rule in max-width: 844px, since if the screen has width <= 632px it also will have width <= 844px. Since time is tight, I'll make a follow-on commit to remove them.